ci/build_container/build_recipes/nghttp2.sh

```diff
@@ -4,8 +4,52 @@ set -e
 
 VERSION=1.32.0
 
-wget -O nghttp2-"$VERSION".tar.gz https://github.com/nghttp2/nghttp2/releases/download/v"$VERSION"/nghttp2-"$VERSION".tar.gz
+curl https://github.com/nghttp2/nghttp2/releases/download/v"$VERSION"/nghttp2-"$VERSION".tar.gz -sLo nghttp2-"$VERSION".tar.gz
 tar xf nghttp2-"$VERSION".tar.gz
 cd nghttp2-"$VERSION"
-./configure --prefix="$THIRDPARTY_BUILD" --enable-shared=no --enable-lib-only
-make V=1 install
+
+# Allow nghttp2 to build as static lib on Windows
+cat > nghttp2_cmakelists.diff << 'EOF'
+diff --git a/lib/CMakeLists.txt b/lib/CMakeLists.txt
+index 17e422b..b2e7a6e 100644
+--- a/lib/CMakeLists.txt
++++ b/lib/CMakeLists.txt
```

Comment:
Are there plans to upstream this?

Reply:
PR: https://github.com/nghttp2/nghttp2/pull/1198